app/(main)/league/[leagueId]/entry/Entries.interface.ts

```diff
@@ -8,13 +8,13 @@ export interface IEntry {
   name: string;
   user: IUser;
   league: ILeague;
-  selectedTeams: INFLTeam[];
+  selectedTeams: INFLTeam['teamName'][];
```

Comment:
Why is `teamName` in quotation marks here? Is it asserting it's type as a string? Or is it going to statically be 'teamName'?

Reply:
@ryandotfurrer this basically tells selected teams that it is of type INFLTeam['teamName] so if that type ever changes it changes everywhere else it is referenced.  So if INFLTeam:

```
Interface INFLTeam {
teamName: string
}
```

['teamName] is just a way to access that specific type reference.